libraries/LittleFS/src/LittleFS.h

```diff
@@ -221,13 +221,48 @@ class LittleFSImpl : public FSImpl
             return false;
         }
 
+        if(_timeCallback && _tryMount()) {
+            // Mounting is required to set attributes
+
+            time_t t = _timeCallback();
+            rc = lfs_setattr(&_lfs, "/", 'c', &t, 8);
+            Serial.printf("rc1: %d\n", rc);
+            if (rc != 0) {
+                DEBUGV("lfs_format, lfs_setattr 'c': rc=%d\n", rc);
+                return false;
+            }
+
+            rc = lfs_setattr(&_lfs, "/", 't', &t, 8);
+            Serial.printf("rc2: %d\n", rc);
+            if (rc != 0) {
+                DEBUGV("lfs_format, lfs_setattr 't': rc=%d\n", rc);
+                return false;
+            }
+            
+            lfs_unmount(&_lfs);
+            _mounted = false;
+        }
+
         if (wasMounted) {
             return _tryMount();
         }
 
         return true;
     }
 
+    time_t creationTime() override {
+        time_t t;
+        uint32_t t32b;
+
+        if (lfs_getattr(&_lfs, "/", 'c', &t, 8) == 8) {
+            return t;
+        } else if (lfs_getattr(&_lfs, "/", 'c', &t32b, 4) == 4) {
+            return (time_t)t32b;
+        } else {
+            return -1;
```

Comment:
Can we return 0 and not -1 for this?  The existing calls return 0 (Jan 1 1970)

Reply:
Same as the other, might make it easier to differentiate between errors van valid values.